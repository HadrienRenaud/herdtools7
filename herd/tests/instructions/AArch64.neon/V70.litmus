AArch64 V70
{
 int x=1;
 0:X0=x; 0:X3=y;
}
P0                    ;
LDR S1,[X3];
FMOV W1,S1            ;
LDR W2,[X0,W1,SXTW]   ;

forall 0:X2=1