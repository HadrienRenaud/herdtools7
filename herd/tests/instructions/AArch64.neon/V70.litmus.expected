Test V70 Required
States 1
0:X2=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X2=1)
Observation V70 Always 1 0
Hash=0e93be9471bfe31656ec91878a8f2670

